lib/services/livesync/platform-livesync-service.ts

```diff
@@ -39,6 +43,10 @@ export abstract class PlatformLiveSyncServiceBase implements IPlatformLiveSyncSe
 			let canExecute = this.getCanExecuteAction(platform, appIdentifier);
 			let action = (device: Mobile.IDevice): IFuture<void> => {
 				return (() => {
+					const normalizePlatformName = this.$mobileHelper.normalizePlatformName(platform);
```

Comment:
Do you think it would be possible to extract some function which can track both events? (build/livesync) Do we have other places where we can track such information?